source/docs/software/labView/creating-robot-programs/creating-building-and-loading-your-benchtop-test-program.rst

```diff
@@ -0,0 +1,56 @@
+Creating, Building and Loading your Benchtop Test Program
+=========================================================
+
+.. image::images/creating-building-and-loading-your-benchtop-test-program/ni-logo.png
```

Comment:
Put a space between the directive double colon and the hyperlink of the directive.

```suggestion
.. image:: images/creating-building-and-loading-your-benchtop-test-program/ni-logo.png
```